### ISSUE TITLE
[a11y][Windows High Contrast Mode]: The close icon of Notification Alert is not visible in Windows High Contrast Mode 

### ISSUE THREAD

**Issue by pnayek** [carbon-design-system/carbon#16268]
### Package

carbon-components, carbon-components-react

### Browser

Chrome

### Operating System

Windows

### Package version

"carbon-components": "10.58.13", "carbon-components-react": "7.59.19"

### Automated testing tool and ruleset

IBM Accessibility Checker tool - Confirm Windows high contrast mode is supported when using CSS to include position, or alter non-decorative content

### Description

The "Close" icon of the Notification Toast component is too dark/invisible in Windows High Contrast Mode.

![image](https://github.com/carbon-design-system/carbon/assets/15056711/21020f38-6285-42ca-8258-ff668d3dbadb)

### WCAG 2.1 Violation

1.1.1 Non-Text Content

### Reproduction/example

N/A

### Steps to reproduce

N/A

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by pnayek** [comment]
Fixed it from our side. Hence closing.
